changelog/fragments/use-operator-lib-metrics.yaml

```diff
@@ -2,18 +2,11 @@
 # release notes and/or the migration guide
 entries:
   - description: >
-      Remove the implementation for `InstrumentedEnqueueRequestForObject`
-      handler from SDK repository and reference it from operator-lib instead.
+      Ansible & Helm-based Operators switched to using the `InstrumentedEnqueueRequestForObject` from
+      [operator-lib](https://github.com/operator-framework/operator-lib/blob/main/handler/instrumented_enqueue_object.go)
+      instead of controller-runtime's `EnqueueRequestForObject`.
```

Comment:
This should be consolidated with the removal of legacy metrics generation in Helm and Ansible operators fragments (again, in the release prep PR)